Add examples to JSONTas documentation (#4)
AxisCommunications/jsontas: docs/examples.rst

            "zipcode": "92998-3874"
        },
        "company": {
            "bs": "harness real-time e-markets",
            "catchPhrase": "Multi-layered client-server neural-net",
            "name": "Romaguera-Crona"
        },
        "email": "Sincere@april.biz",
        "id": 1,
        "name": "Leanne Graham",
        "phone": "1-770-736-8031 x56442",
        "username": "Bret",
        "website": "hildegard.org"
      }
   }

Getting a specific response from the response will be further explained below in segment Nested_

Wait
----

:obj:`jsontas.data_structures.wait`

Waiting for a query tree to evaluate to true. A query tree is the full, uresolved, JSON structure that is currently being resolved.
This is mostly used for when utilizing the Request_ datastructure as a way to wait for an API to respond with the data required.

Example will look similar to the Request_ example as this is how the Wait_ structure is normally used.

Dataset
^^^^^^^

.. code-block:: json

   {
      "userdata": "https://jsonplaceholder.typicode.com/users/this_does_not_exist"
   }

JSON
^^^^

.. code-block:: json

   {
      "user": {
         "$wait": {
            "for": {

>                "request": {

Reviewer: ```suggestion
               "$request": {
```

                  "url": "$userdata",
                  "method": "GET"
               },
               "interval": 1,
               "timeout": 5,

>                "else": "No user found"

Reviewer: move outside for

      },
      "posts": {
         "$from": {
            "item": {
               "$request": {
                  "url": "$posts_api",
                  "method": "GET",
                  "params": {
                     "userId": "$this.user_id.0"
                  }
               }
            },
            "get": "title"
         }
      }
   }

Result
^^^^^^

.. code-block:: json

   {
      "posts": [
         "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
         "qui est esse",
         "ea molestias quasi exercitationem repellat qui ipsa sit aut",
         "eum et est occaecati",
         "nesciunt quas odio",
         "dolorem eum magni eos aperiam quia",
         "magnam facilis autem",
         "dolorem dolore est ipsam",
         "nesciunt iure omnis dolorem tempora et accusantium",
         "optio molestias id quia eum"
      ],
      "user_id": [
          1
      ]
   }

Conclusion
----------

There are many crazy ways of utilizing JSONTas and it's quite impossible to write examples for each and every use-case.
If there are any questions then please to not hestitate contacting the maintainers or writing an issue in github.

> Or just play around with it and send a PR with new examples.

Reviewer: ```suggestion
We encurage you to play around with it and send a PR with new examples.
```
